DoePaAdminViewModel/Services/DoePaAdminService.cs

```diff
@@ -108,6 +110,35 @@ public async Task<Anstellungsdetail> CreateAnstellungsdetailAsync(CancellationTo
             return await AddDataToDbSetAsync(DBContext.Anstellungsdetails, cancellationToken);
         }
 
+        public async Task<IEnumerable<EmployeeInvoicedHours>> GetEmployeeInvoicedHoursAsync(string email, DateTime from, DateTime to, CancellationToken cancellationToken = default)
+        {
+            var result = DBContext
+                .Ausgangsrechnungspositionen
+                        .Include(arp => arp.ZugehoerigeRechnung)
+                        .Include(arp => arp.ZugehoerigeKostenstelle)
+                        .Include(arp => arp.ZugehoerigeAuftragsposition)
+                            .ThenInclude(zap => zap.Auftrag)
+                                .ThenInclude(a => a.ZugehoerigesProjekt)
+                                    .ThenInclude(p => p.Rechnungsempfaenger)
+                                        .ThenInclude(r => r.ZugehoerigerKunde)
+                        .Include(arp => arp.ZugehoerigeAuftragsposition.Auftrag.VerantwortlicherMitarbeiter)
+                        .Include(arp => arp.ZugehoerigeAbrechnungseinheit)
+                    .Where(arp => arp.ZugehoerigeAuftragsposition.Auftrag.VerantwortlicherMitarbeiter.Email == email)
+                    .Where(arp => arp.ZugehoerigeAbrechnungseinheit.Name.Equals("Stunden"))
+                    .Where(arp => arp.LeistungszeitraumVon <= to)
+                    .Where(arp => arp.LeistungszeitraumBis >= to)
```

Comment:
I think we need to turn this query around here. Something like this:
`.Where(arp => arp.LeistungszeitraumBis >= from).Where(arp => arp.LeistungszeitraumBis <= to)`

Reply:
Yes, I had already thought about that